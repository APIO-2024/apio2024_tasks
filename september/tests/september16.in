txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 5
0 0 1 2 3 5
3 4 5 2 6 1
5 3 1 4 2 6
5 6 2 3 1 4
4 2 1 6 3 5
1 2 6 3 5 4
10 5
0 1 2 3 4 4 5 7 8
7 9 6 5 8 4 3 2 1
9 7 5 6 8 4 2 3 1
9 5 6 8 7 4 2 3 1
8 7 9 6 5 4 2 3 1
6 8 5 7 9 4 1 3 2
7 5
0 0 1 3 3 5
4 6 5 3 1 2
4 6 5 3 1 2
6 4 3 5 1 2
6 4 5 3 1 2
4 6 3 5 1 2
6 5
0 1 2 3 4
4 3 2 5 1
1 5 2 4 3
1 3 4 5 2
5 3 1 4 2
1 2 4 5 3
