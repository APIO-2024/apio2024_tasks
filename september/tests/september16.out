7ckgnn4wyi495puj3ibqf81dqvapyv6b
1
3
4
1
